### DISCUSSION TITLE
Container to "cast" to?

### DISCUSSION THREAD

**Discussion by Flirre** [tteck/Proxmox#1838] [Q&A]
I've been using librespot to get a "target" for my speakers which is working great as long as I just want to listen to Spotify. 

But I also want to be able to listen to music on/from Youtube (and perhaps other sources?). 
So I'm wondering if there is any application to get like a "chromecast-target" or similar?

**Comment by lucas-0liveira** [comment]
Yes, there is a solution for that. You can consider using an application called Stream What You Hear (SWYH). It allows you to stream audio from your PC to compatible devices, functioning similarly to a "chromecast-target" for audio. With SWYH, you can stream audio from different sources, including YouTube and other sources, to your speakers. This provides a versatile solution for listening to music from various sources on your audio devices.
